src/seedu/addressbook/Main.java

```diff
@@ -55,7 +56,7 @@ private void start(String[] launchArgs) {
             this.addressBook = storage.load();
             ui.showWelcomeMessage(VERSION, storage.getPath());
 
-        } catch (InvalidStorageFilePathException | StorageOperationException e) {
+        } catch (InvalidStorageFilePathException | StorageOperationException | FileNotFoundException e) {
```

Comment:
You can either show the error and terminate the program or create a file and continue the program. 

start() is not the place to catch this exception because the file was deleted after the initial load. 

The best place to catch is Main::executeCommand() (this method already has a catch block to terminate upon exception.